src/wallet/rpcdump.cpp

```diff
@@ -375,6 +375,7 @@ UniValue importprunedfunds(const JSONRPCRequest& request)
                 {
                     {"rawtransaction", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "A raw transaction in hex funding an already-existing address in wallet"},
                     {"txoutproof", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The hex output from gettxoutproof that contains the transaction"},
+                    {"height", RPCArg::Type::NUM, RPCArg::Optional::NO, "The height in which transaction has been confirmed"},
```

Comment:
Is this required? the txoutproof includes a block header, which could be used to look up the height in the chain.

Reply:
Hmmm well we could use `getBlockHeight` again in this function but I didn't want because was planning to remove this Chain method after we move ScanForWalletTransactions and its logic on the node side.

Maybe that a bit premature, but IMO this RPC should come with all proofs + height without us having to query the chain..

Reply:
I don't like this change for a couple of reasons:

- it's a breaking change for clients, so they'll need to reimplement their code
- the RPC doesn't do any checking on the `height` value passed, so it's possible to get the wallet into some inconsistent state by passing a bad `height` value.